extensions/control-plane/provision/provision-blob/src/main/java/org/eclipse/edc/connector/provision/azure/AzureProvisionExtension.java

```diff
@@ -35,6 +36,9 @@
  */
 public class AzureProvisionExtension implements ServiceExtension {
 
+    @Setting(description = "Expiration time, in hours, for the SAS token.", defaultValue = "1", required = false)
+    public static final String EDC_AZURE_TOKEN_EXPIRY_TIME = "edc.azure.token.expiry.time";
```

Comment:
setting injection can be used here as well, also because I see a discrepancy between the documentation (there's a default value) and the code (like 67: no default value provided).

Reply:
Good point, added the `AzureProvisionConfiguration` in [here](https://github.com/eclipse-edc/Technology-Azure/pull/298/commits/a20a5bd83a4033c9249b24ca810cbacdc8a248df) and, despite only having one setting, follows the best practice as well. Config bean is then passed down.